pyvista/core/utilities/geometric_objects.py

```diff
@@ -308,37 +309,48 @@ def Sphere(
 ):
     """Create a sphere.
 
+    A sphere describes a 2D surface in comparison to
+    :func:`pyvista.SolidSphere`, which fills a 3D volume.
+
+    pv uses a convention where ``theta`` represents the azimuthal
```

Comment:
Was this part of a bulk change to use `pv` for importing?  I think it doesn't read here as well as `pv` rather than `PyVista`.

Reply:
Yup, part of https://github.com/pyvista/pyvista/pull/4965/commits/691ebc6c51f579ca492bb9f5fdf2eb6742b3bc48. Seems like a mistake, along with some other similar ones.

Reply:
Thanks for catch. It's my fault.
```suggestion
    PyVista uses a convention where ``theta`` represents the azimuthal
```